modules/flake/overlays.nix

```diff
@@ -2,15 +2,15 @@
 {
   # Provide overlay to add `nix-snapshotter`.
   flake.overlays.default = self: super: {
-    nix-snapshotter = self.callPackage ../../package.nix {};
+    nix-snapshotter = self.callPackage ../../package.nix { };
 
-    # Depends on PR merged into main, but not yet in a release tag.
-    # See: https://github.com/containerd/containerd/pull/9028
-    containerd = super.containerd.overrideAttrs(o: {
-      src = self.fetchFromGitHub {
-        inherit (o.src) owner repo;
-        rev = "779875a057ff98e9b754371c193fe3b0c23ae7a2";
-        hash = "sha256-sXMDMX0QPbnFvRYrAP+sVFjTI9IqzOmLnmqAo8lE9pg=";
```

Comment:
I smoked test the latest changes but k3s wouldn't come up healthy. Also, I am wondering why `containerd` has been removed here? cc @RobbieBuxton

Reply:
Sorry, I was being silly there I shouldn't have removed that. I thought I had forgotten to remove that when upgrading to "23.11" in my last PR but the required fix isn't actually included in the current version on stable ```containerd 1.7.11```. I'll revert that now